tools/importers/SynchronizeTypographyTokens.ts

```diff
@@ -173,29 +187,33 @@ function buildTypographyTokensMixins(tokens: Array<ITypographyToken>): Array<str
 
 const SynchronizeTypographyTokensBin = async () => {
 	console.log('Import in progress...');
-	let requestResponse = await requestForBin(tokensFilesConfig.bin);
+	const tokensFilesConfig = require('./configs/SynchronizeTypographyTokensConfig.json');
+	let requestResponse = await requestForBin(
+		tokensFilesConfig.bin,
+		tokensFilesConfig.jsonBinApiUrl,
+	);
 
 	ImportTypographyVariables(
-		tokensFilesConfig.bin.files,
+		tokensFilesConfig,
 		requestResponse.data.record.values.LMSDesignSystemTypography,
 	);
 
 	ImportTypographyTokens(
-		tokensFilesConfig.bin.files,
+		tokensFilesConfig,
 		requestResponse.data.record.values.LMSDesignSystemTypography,
 	);
 
 	console.log('The import was successful for bin: ' + tokensFilesConfig.bin.id);
 };
 
-const requestForBin = async (binConfig: TypographyConfigFileBin) => {
+export const requestForBin = async (binConfig: TypographyConfigFileBin, jsonBinApiUrl: string) => {
```

Comment:
are you sure that we should export this function? Does it makes sense for devs to call it separately from the rest of the code?

Reply:
It's reverted.